src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -298,6 +298,20 @@ export class BuildGraphTableComponent implements OnChanges {
     if ('graph' in changes && this.graph) {
       this.sortedBuilds = sortBuilds(this.graph);
     }
+
+    if(changes.showAllDependencies && (typeof changes.showAllDependencies.previousValue !== 'undefined') && (changes.showAllDependencies.previousValue != changes.showAllDependencies.currentValue))
+    {
+      let showSubDependenciesSetting = changes.showAllDependencies.currentValue ? "on" : "off";
+
+      this.ai.trackEvent({name: "Show Sub-Dependencies"}, {value: showSubDependenciesSetting});
```

Comment:
>value [](start = 59, length = 5)

I'd call the "value" something more descriptive. All the evens get mixed together in the events table, so it might tricky to find the value with the name "value". :-) Maybe "optionEnabled" or something?